pkg/errors/reexport.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errors
+
+import (
+	"errors"
+
+	perrors "github.com/pingcap/errors"
```

Comment:
After this PR, there is only one import path `github.com/pingcap/tiflow/errors`. We will not to use go errors directly.

Reply:
On the other hand, it's hard to unify the alias names inside errors packages, since we are importing `github.com/pingcap/errors` without an alias to define our errors. See https://github.com/pingcap/tiflow/blob/master/pkg/errors/engine_errors.go#L30. If we unified the alias like the above, there would be a lot of changes.

Reply:
This is reasonable, and we could change the alias if needed in the future .